From: Mel Gorman <mgorman@suse.de>
Date: Tue, 2 Oct 2018 13:21:28 +0100
Subject: [PATCH] mm, numa: Remove rate-limiting of automatic numa balancing
 migration kabi

References: bnc#1101669 optimise numa balancing for fast migrate
Patch-mainline: Never, kabi

Preserve KABI of struct pglist_data.

[js] removed by commit efaffc5e40aeced0bcb497ed7a0a5b8c14abfcdf.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 include/linux/mmzone.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/linux/mmzone.h b/include/linux/mmzone.h
index 39d74cac0827..10928d087fe0 100644
--- a/include/linux/mmzone.h
+++ b/include/linux/mmzone.h
@@ -672,6 +672,10 @@ typedef struct pglist_data {
 #ifdef CONFIG_NUMA_BALANCING
 	/* Lock serializing the migrate rate limiting window */
 	spinlock_t numabalancing_migrate_lock;
+
+	/* Dead fields preserved for KABI */
+	unsigned long numabalancing_migrate_next_window;
+	unsigned long numabalancing_migrate_nr_pages;
 #endif
 	/*
 	 * This is a per-node reserve of pages that are not available
